### PR TITLE
Helm chart log-level extension

### PR DESCRIPTION
**Pull request by mrahbar**
added log-level env in rook-operator deployment.yaml with variable .Values.logLevel

I hope this PR is complementary to #938

### CODE REVIEW

File: cluster/charts/rook/templates/deployment.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@ spec:
         env:
         - name: ROOK_REPO_PREFIX
           value: {{ .Values.image.prefix }}
+        - name: ROOK_LOG_LEVEL
```
Comment (kokhang) [inline cluster/charts/rook/templates/deployment.yaml]: where is this env being consumed?

Comment (travisn) [inline cluster/charts/rook/templates/deployment.yaml]: the operator would pick it up in the --log-level command line arg

**Comment by jbw976** [comment]
The documentation at https://github.com/rook/rook/blob/master/Documentation/helm-operator.md should also be updated in this PR.

**Comment by mrahbar** [comment]
Sorry but i messed up the commit squash, can you perform it when merging please

**Comment by travisn** [comment]
@mrahbar instead of merging, could you try rebasing as described in this?
 https://github.com/rook/rook/blob/master/Documentation/development-flow.md#updating-your-fork 
Assuming you have the upstream remote setup, the basic commands are 
```
git fetch --all
git rebase upstream/master
```
If that doesn't work to clean up the history, you may need to hard reset the merge commit and then try the rebase again.

**Pull request merged**